lib/trento_web/channels/user_channel.ex

```diff
@@ -0,0 +1,34 @@
+defmodule TrentoWeb.UserChannel do
+  @moduledoc """
+  User channel, each use is subscribed to his channel,
+  to receive personal broadcasts
+
+  Users can't join other users channels.
```

Comment:
Better `Users can't join other users channel`.
And I don't really understand well the next phrase `This has a prerequisite...`

Reply:
adjusted!